CenterForOpenScience/osf.io
# [PLAT-693] Make embedded wiki images viewable through a view-only link

## Purpose

Currently using a view-only link will break images or other files embedded in the wiki. This PR adds the view_only param into the embedded file links allowing the file to be viewed.

## Changes

- simple JS changes to the grab the image links and append the view_only param
- adds a new conditional to the markdown to add the view_only param to the embedded files.

## QA Notes

Create Wiki with various embedded files in a private project, then create a view-only link and try to view those files while logged out with the view-only link. You should be able to see them now.

Also remember to use the various sizing options available! 

## Documentation

🐞 fix, no docs.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/PLAT-693
